projects/ntpd-rs/default.nix

```diff
@@ -0,0 +1,10 @@
+{
+  pkgs,
+  lib,
+  sources,
+} @ args: {
+  packages = {inherit (pkgs) ntpd-rs;};
+  nixos = {
+    modules.services.ntpd-rs = null;
```

Comment:
```suggestion
    modules.services.ntpd-rs = "${sources.inputs.nixpkgs}/nixos/modules/services/networking/ntp/ntpd-rs.nix";
```

Reply:
As suggested by @stepbrobd over in #305. Also, #305 will need to be merged first, since it includes changes to the flake.nix that are needed for this to have direct access to the nixpkgs input.